modules/index/src/main/java/org/apache/ignite/internal/index/SortedIndex.java

```diff
@@ -39,24 +41,28 @@ public interface SortedIndex extends Index<SortedIndexDescriptor> {
     byte INCLUDE_RIGHT = 0b10;
 
     /**
-     * Opens a range cursor for given bounds with left bound included in result and right excluded.
+     * Opens a range cursor for given bounds. Inclusion of the bounds is defined by {@code includeBounds} mask.
```

Comment:
what **includeBounds** are you talking about ? can u explain ?

Reply:
Thanks, fixed.
> can u explain ?
This is just a copy-paste from the methods below) I fixed javadoc for them too.